notebooks/primary-care-covid-codes.ipynb

```diff
@@ -232,299 +84,120 @@
-    "        'Record of contact/exposure/procedure',\n",
-    "        'Record of negative test result for SARS-CoV-2',\n",
-    "        'Healthcare contact related to COVID-19 but not case status',      \n",
-    "    ],\n",
-    "    'link':[\n",
-    "        \"https://codelists.opensafely.org/codelist/opensafely/covid-identification-in-primary-care-probable-covid-clinical-code/2020-07-16/\",\n",
-    "        \"https://codelists.opensafely.org/codelist/opensafely/covid-identification-in-primary-care-probable-covid-positive-test/2020-07-16/\",\n",
-    "        \"https://codelists.opensafely.org/codelist/opensafely/covid-identification-in-primary-care-probable-covid-sequelae/2020-07-16/\",\n",
-    "        \"https://codelists.opensafely.org/codelist/opensafely/covid-identification-in-primary-care-suspected-covid-advice/2020-07-16/\",\n",
-    "        \"https://codelists.opensafely.org/codelist/opensafely/covid-identification-in-primary-care-suspected-covid-had-test/2020-07-16/\",\n",
-    "        \"https://codelists.opensafely.org/codelist/opensafely/covid-identification-in-primary-care-suspected-covid-isolation-code/2020-07-16/\",\n",
-    "        \"https://codelists.opensafely.org/codelist/opensafely/covid-identification-in-primary-care-suspected-covid-nonspecific-clinical-assessment/2020-07-16/\",\n",
-    "        \"https://codelists.opensafely.org/codelist/opensafely/covid-identification-in-primary-care-suspected-codes-suspected-codes/2020-07-16/\",\n",
-    "        \"https://codelists.opensafely.org/codelist/opensafely/covid-identification-in-primary-care-historic-case/2020-06-23/\",\n",
-    "        \"https://codelists.opensafely.org/codelist/opensafely/covid-identification-in-primary-care-potential-historic-case/2020-06-23/\",\n",
-    "        \"https://codelists.opensafely.org/codelist/opensafely/covid-identification-in-primary-care-exposure-to-disease/2020-06-23/\",\n",
-    "        \"https://codelists.opensafely.org/codelist/opensafely/covid-identification-in-primary-care-antigen-test-negative/2020-06-24/\",\n",
-    "        \"https://codelists.opensafely.org/codelist/opensafely/covid-identification-in-primary-care-unrelated-to-case-status/2020-06-23/\",\n",
-    "    ]\n",
-    "}\n",
-    "\n",
-    "tabledata = pd.DataFrame(tabledata, index=tableindex)\n",
-    "tabledata['Codelist']=\"<a href='\"+tabledata['link']+\"' target='_blank'>\"+tabledata['Codelist']+\"</a>\"\n",
-    "\n",
-    "codecounts_total.name = \"Count\"\n",
-    "\n",
-    "tabledata = tabledata.merge(codecounts_total, left_index=True, right_index=True)\n",
-    "\n",
-    "# easy but not ideal output\n",
-    "#display(HTML(tabledata.to_html(index=False, justify='left')))\n",
-    "\n",
-    "# use styling instead - https://pandas.pydata.org/pandas-docs/stable/user_guide/style.html\n",
+    "os.getcwd()\n",
     "styles = [dict(selector=\"th\", props=[(\"text-align\", \"left\")])]\n",
+    "tabledata = pd.read_csv(f\"../output/tabledata.csv\")\n",
```

Comment:
I will have to update all file paths to the released outputs once this  has been run.